app/mailers/report_mailer.rb

```diff
@@ -19,4 +19,14 @@ def publication_results_email(results)
          cc: ENV['MAINTAINER_EMAIL'],
          subject: 'DSpace publication results summary')
   end
+
+  def preservation_results_email(results)
+    return unless ENV.fetch('DISABLE_ALL_EMAIL', 'true') == 'false' # allows PR builds to disable emails
```

Comment:
This still breaks my brain every time I see it. I still can't parse what it does, but only know it works because it works.